app/handlers/exceptions_app.rb

```diff
@@ -0,0 +1,53 @@
+class ExceptionsApp < Rambulance::ExceptionsApp
+  before_action :set_locals
+
+  def bad_request
+    render :generic
```

Comment:
can we refactor this to avoid duplication and have

```ruby
after_action :render_generic, only: [:bad_request, :conflict, :method_not_allowed, :not_acceptable, :not_implemented]

def render_generic
  render :generic
end
```
?
or similar approach

Reply:
Ah - thanks for catching that. I used an [exception handler](https://github.com/SUSE-Enceladus/suse-rancher-setup/blob/cf0ce3e0c8032d7b58beb07d3ddcbfced5eeaba4/app/handlers/exceptions_app.rb#L50-L52) for that, but forgot to remove the redundant `render` lines.